src/providers/CourtConfig.js

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react'
+import PropTypes from 'prop-types'
+import { useSubscription } from 'urql'
+import { CourtConfig as CourtConfigSubscription } from '../queries/courtConfig'
+
+import environment from '../environment'
+import { getNetworkName } from '../web3-utils'
+import { networks } from '../networks'
+
+const CHAIN_ID = environment('CHAIN_ID')
+const CourtConfigContext = React.createContext()
+
+function CourtConfigProvider({ children }) {
+  const courtAddress = networks[getNetworkName(CHAIN_ID)].court
+
+  console.log(courtAddress)
+
+  const [result] = useSubscription({
+    query: CourtConfigSubscription,
+    variables: { id: courtAddress.toLocaleLowerCase() },
+  })
+
+  // TODO: handle possible errors
+  const courtConfig = result.data && result.data.courtConfig
+
+  return (
+    <CourtConfigContext.Provider value={courtConfig}>
```

Comment:
should we pass all the config or one prop by prop?

Reply:
Unless we need to do some conversion i think that is ok to pass directly the configuration. Is there a reason that you think would be good to pass it like you mention?